features/restaurant_index.feature

```diff
@@ -0,0 +1,8 @@
+Feature: The welcome page
+  As a User
+  in order to order food
+  I need to visit a landing page
+
+  Scenario: User visits the site
+    Given I navigate to the index page
+    Then I should see welcome message "What do you feel like tonight?"
```

Comment:
Why do you choose to scope this step so specifically? I'm thinking about the wording "welcome message ". Why not keep it more generic?

Reply:
because we were thinking specifically about this message and forgot to think about we would reuse this line.